redisinsight/ui/src/pages/settings/SettingsPage.tsx

```diff
@@ -62,12 +61,20 @@ const SettingsPage = () => {
     })
   }, [])
 
+  useEffect(() => {
+    if (config) {
+      setThemeValue(config.theme);
+      theme = config.theme;
+    }
+  }, [config])
+
```

Comment:
Let's extract the entire theme setup into a separate component, similar to `<ConsentsPrivacy />` or `<WorkbenchSettings />`. This will help keep the SettingsPage cleaner and may also reduce unnecessary re-renders.

Reply:
added a new component `<ThemeSettings />`